docs/integrations/saas-cloud/knowbe4.md

```diff
@@ -0,0 +1,154 @@
+| first(status) as status group by pst_id, name
+| where status matches "{{status}}"
+| count_distinct(pst_id)
+```
+
+
+```sql title="Total Recipient User"
+_sourceCategory="knowbe4nfr" recipient_id
+| json "recipient_id", "pst_id", "user.first_name", "user.last_name", "user.email", "scheduled_at", "delivered_at", "opened_at", "clicked_at", "replied_at", "attachment_opened_at", "macro_enabled_at", "data_entered_at", "qr_code_scanned", "reported_at", "bounced_at", "ip", "ip_location", "browser", "browser_version", "os" as recipient_id, pst_id, first_name, last_name,
+email, scheduled_at, delivered_at, opened_at, clicked_at, replied_at, attachment_opened_at, macro_enabled_at, data_entered_at, qr_code_scanned, reported_at, bounced_at, ip, ip_location, browser, browser_version, os nodrop
+| where ip_location matches "{{user_location}}"
+| count_distinct(recipient_id)
+```
+
+## Installing the KnowBe4 App
+
+Locate and install the app from the **App Catalog**. To see a preview of the dashboards included with the app before installing, click **Preview Dashboards**.
+Before you begin, collect logs from KnowBe4 and ingest them into Sumo Logic. Refer to the [KnowBe4 Cloud-to-Cloud Integration](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/knowbe4-api-source/) to create the source and use the same source category while installing the app.
+
+To install the app, follow the steps below:
+1. From the **App Catalog**, search for the app and select it.
+1. Select **Add Integration** button to install the app.
+1. Configure **KnowBe4 App** using the steps described in the [KnowBe4 Cloud-to-Cloud Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/knowbe4-api-source/). If you already have set up your data, skip this step by clicking on **Next**.
+1. Complete the following fields:
+   1. **Data Source**. Select either of these options for the data source:
+      * Choose **Source Category** and then choose a source category from the list.
+      * Select **Enter a Custom Data Filter** and type in a custom source category that starts with an underscore. For example, `_sourceCategory=MyCategory`.
+    2. **Folder Name**. You can retain the existing name, or enter a name of your choice for the app. 
+    3. Select the **Location in Library** (the default is the **Personal** folder in the library), or click **New Folder** to add a new folder.
+1. Click **Next**.
+
+Once an app is installed, it will appear in your **Personal** folder, or other folder that you specified. You can share it with your organization.
+
+The panels will begin to fill automatically. It's worth noting that each panel gradually fills with data that matches the time range query and has been received since the panel was created. The results will not be available right away, but with some patience, you will be able to view full graphs and maps.
```

Comment:
```suggestion
The panels will begin to fill automatically. It's worth noting that each panel gradually fills with data that matches the time range query and has been received since the panel was created. The results will not be available right away, but you will be able to view full graphs and maps.
```

Reply:
Updated the code.